pkg/chunked/storage_linux.go

```diff
@@ -230,21 +230,21 @@ func makeEstargzChunkedDiffer(ctx context.Context, store storage.Store, blobSize
 func copyFileFromOtherLayer(file *internal.FileMetadata, source string, otherFile *internal.FileMetadata, dirfd int, useHardLinks bool) (bool, *os.File, int64, error) {
 	srcDirfd, err := unix.Open(source, unix.O_RDONLY, 0)
 	if err != nil {
-		return false, nil, 0, err
+		return false, nil, 0, errors.Wrapf(err, "open source file %q", source)
```

Comment:
This looks like a stutter.

Reply:
the `unix.Open` call doesn't include any extra information except the errno